README.md

```diff
@@ -1,105 +1,301 @@
+## Installation
 
-    `options.serializers` _(List[Object])_: A list with serializers to apply. Each serializers must contain two properties: `path` (path for the value to be serialized, must be a `string`) and `serializer` (function to be called on the path's value).
+```bash
+npm install @uphold/anonymizer
+```
 
-    `options.trim` _(Boolean)_: A flag that enables trimming all redacted values, saving their keys to a `__redacted__` list (default value is `false`).
+## Usage
 
-### Example
+### Basic example
 
 ```js
-const { anonymizer } = require('@uphold/anonymizer');
-const whitelist = ['foo.key', 'foo.depth.*', 'bar.*', 'toAnonymize.baz', 'toAnonymizeSuperString'];
-const blacklist = ['foo.depth.innerBlacklist', 'toAnonymize.*'];
+import { anonymizer } from '@uphold/anonymizer';
+
+const whitelist = ['key1', 'key2.foo'];
+const anonymize = anonymizer({ whitelist });
+
+const data = {
+  key1: 'bar',
+  key2: {
+    foo: 'bar',
+    bar: 'baz',
+    baz: {
+      foo: 'bar',
+      bar: 'baz'
+    }
+  }
+};
+
+anonymize(data);
+
+// {
+//   key1: 'bar',
+//   key2: {
+//     foo: 'bar',
+//     bar: '--REDACTED--',
+//     baz: {
+//       foo: '--REDACTED--'
+//       bar: '--REDACTED--'
+//     }
+//   }
+// }
+```
+
+### Wildcard matching example
+
+Using `*` allows you to match any character in a key, except for `.`.
+This is similar to how `glob` allows you to use `*` to match any character, except for `/`.
+
+```js
+import { anonymizer } from '@uphold/anonymizer';
+
+const whitelist = ['key2.*'];
+const anonymize = anonymizer({ whitelist });
+
+const data = {
+  key1: 'bar',
+  key2: {
+    foo: 'bar',
+    bar: 'baz',
+    baz: { foo: 'bar' }
+  }
+};
+
+anonymize(data);
+
+// {
+//   key1: '--REDACTED--',
+//   key2: {
+//     foo: 'bar',
+//     bar: 'baz',
+//     baz: {
+//       foo: '--REDACTED--'
+//       bar: '--REDACTED--'
+//     }
+//   }
+// }
+```
+
+### Double wildcard matching example
+
+Using `**` allows you to match any nested key.
+This is similar to how `glob` allows you to use `**` to match any nested directory.
+
+```js
+import { anonymizer } from '@uphold/anonymizer';
+
+const whitelist = ['key2.**', '**.baz'];
+const blacklist = ['key2.bar']
 const anonymize = anonymizer({ blacklist, whitelist });
 
 const data = {
-  foo: { key: 'public', another: 'bar', depth: { bar: 10, innerBlacklist: 11 } },
-  bar: { foo: 1, bar: 2 },
-  toAnonymize: { baz: 11, bar: 12 },
-  toAnonymizeSuperString: 'foo'
+  key1: 'bar',
+  key2: {
+    foo: 'bar',
+    bar: 'baz',
+    baz: { foo: 'bar' }
+  },
+  key3: {
+    foo: {
+      baz: 'biz'
+    }
+  }
 };
 
 anonymize(data);
 
 // {
-//   foo: {
-//     key: 'public',
-//     another: '--REDACTED--',
-//     depth: { bar: 10, innerBlacklist: '--REDACTED--' }
+//   key1: '--REDACTED--',
+//   key2: {
+//     foo: 'bar',
+//     bar: '--REDACTED--',
+//     baz: {
+//       foo: 'bar'
+//       bar: 'baz'
```

Comment:
Same as https://github.com/uphold/anonymizer/pull/83/files#r1865645161

Reply:
Added it to the example data.